papaslib/CMakeLists.txt

```diff
@@ -1,24 +1,11 @@
 cmake_minimum_required(VERSION 2.8)
 
 # use, i.e. don't skip the full RPATH for the build tree
-SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
+SET(CMAKE_SKIP_BUILD_RPATH  TRUE)
```

Comment:
I think the RPATH may be there for historical reasons. I tested on both Mac and lxplus and it seems OK without, so I have removed it. If I turn out to be mistaken I can make it operating system dependent.